pkg/apis/vsphereprovider/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -106,7 +106,6 @@ func (in *VSphereMachineProviderSpec) DeepCopyObject() runtime.Object {
 func (in *VSphereMachineProviderStatus) DeepCopyInto(out *VSphereMachineProviderStatus) {
 	*out = *in
 	out.TypeMeta = in.TypeMeta
-	in.ObjectMeta.DeepCopyInto(&out.ObjectMeta)
```

Comment:
:/?

Reply:
It's the output of `go generate` and I removed `ObjectMeta` from status